class.jetpack-options.php

```diff
@@ -153,13 +183,28 @@ public static function get_option( $name, $default = false ) {
 	 * @param string $name Option name
 	 * @param mixed $default (optional)
 	 *
-	 * @return mixed|void
+	 * @return mixed
 	 */
 	public static function get_option_and_ensure_autoload( $name, $default ) {
-		$value = get_option( $name );
+		// In this function the name is not adjusted by prefixing jetpack_
```

Comment:
Probably should indicate in the various functions in the phpdoc whether it expects the prefixed or unprefixed option names.  Possibly evaluate changing the ones using prefixed versinos to private member functions so there's a consistent usage for all public-facing methods?